lib/shopify_app/webhooks_manager_job.rb

```diff
@@ -0,0 +1,11 @@
+module ShopifyApp
+  class WebhooksManagerJob < ActiveJob::Base
+    def perform(params = {})
+      shop_name = params[:shop_name]
```

Comment:
`params.fetch(:shop_name)` and `params.fetch(:token)` would provide slightly better error messages if one of those keys is not provided for some reason.